pkg/apis/abac/register.go

```diff
@@ -43,7 +45,7 @@ var (
 )
 
 func addKnownTypes(scheme *runtime.Scheme) error {
-	scheme.AddKnownTypes(unversioned.GroupVersion{Group: Group, Version: runtime.APIVersionInternal},
+	scheme.AddKnownTypes(SchemeGroupVersion,
```

Comment:
This is how we've done it all register files since this list can get pretty long.